SPECS/libbpf/libbpf.spec

```diff
@@ -0,0 +1,125 @@
+Summary:        Libbpf library
+Name:           libbpf
+Version:        0.4.0
+Release:        3%{?dist}
+License:        LGPLv2 OR BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/%{name}/%{name}
+#Source0:       https://github.com/%{name}/%{name}/archive/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+BuildRequires:  elfutils-devel
+BuildRequires:  elfutils-libelf-devel
+BuildRequires:  gcc
+BuildRequires:  make
+
+%description
+A mirror of bpf-next linux tree bpf-next/tools/lib/bpf directory plus its
+supporting header files. The version of the package reflects the version of
+ABI.
+
+%package        devel
+Summary:        Development files for %{name}
+Requires:       %{name} = %{version}-%{release}
+Requires:       kernel-headers >= 5.9.0
+Requires:       zlib
+
+%description    devel
+The %{name}-devel package contains libraries header files for
+developing applications that use %{name}
+
+%package        static
+Summary:        Static library for libbpf development
+Requires:       %{name}-devel = %{version}-%{release}
+
+%description static
+The %{name}-static package contains static library for
+developing applications that use %{name}
+
+%define _lto_cflags %{nil}
+%global make_flags DESTDIR=%{buildroot} OBJDIR=%{_builddir} CFLAGS="%{build_cflags} -fPIC" LDFLAGS="%{build_ldflags} -Wl,--no-as-needed" LIBDIR=/%{_libdir} NO_PKG_CONFIG=1
+
+%prep
+%autosetup
+
+%build
+%make_build -C ./src %{make_flags}
+
+%install
+%make_install -C ./src %{make_flags}
+
+%files
+%{_libdir}/libbpf.so.%{version}
+%{_libdir}/libbpf.so.0
+
+%files devel
+%{_libdir}/libbpf.so
+%{_includedir}/bpf/
+%{_libdir}/pkgconfig/libbpf.pc
+
+%files static
```

Comment:
Can we also remove the static libs? If we don't need them, we should avoid providing them to not tempt anyone.;)

Reply:
Thanks- checked that no pending packages depend on the static subpackage, so I went ahead and removed it.